<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="chatwidget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#chatwidget{
	border-image: url(:/img/chat_Img/chatframe2.png);
}</string>
   </property>
   <widget class="QPushButton" name="words4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/chat_Img/chatframe3-1.png.png);</string>
    </property>
    <property name="text">
     <string>OI!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="words1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/chat_Img/chatframe3-1.png.png);</string>
    </property>
    <property name="text">
     <string>hello</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="words3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/chat_Img/chatframe3-1.png.png);</string>
    </property>
    <property name="text">
     <string>meowwwwww</string>
    </property>
   </widget>
   <widget class="QPushButton" name="words2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Demi ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/chat_Img/chatframe3-1.png.png);</string>
    </property>
    <property name="text">
     <string>goodbye</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshbtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/chat_Img/refresh.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="responselabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/chat_Img/chatframe3.png);</string>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <widget class="QLabel" name="responsetext">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Demi ITC</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>meowwww~~~</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Cat">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>140</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/feeding_Img/miao.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>Cat</zorder>
  <zorder>chatwidget</zorder>
  <zorder>responselabel</zorder>
  <zorder>responsetext</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
